app/components/conversations/conversation-thread.js

```diff
@@ -1,9 +1,14 @@
 import Component from '@ember/component';
 import { run } from '@ember/runloop';
+import EmberCan from 'ember-can';
+import { alias } from '@ember/object/computed';
 
 export default Component.extend({
   classNames: ['conversation-thread'],
 
+  ability: EmberCan.computed.ability('project'),
+  canAdminister: alias('ability.canAdminister'),
```

Comment:
You're using `{{#if  can 'administer' 'project'}}` below, so these two lines are no longer necessary.